gr-fec/lib/awgn_bp.cc

```diff
@@ -78,7 +83,7 @@ void awgn_bp::rx_lr_calc(std::vector<float> codeword)
     float y;
     for (int i = 0; i < N; i++) {
         y = codeword[i];
-        rx_lr[i] = exp((-1 * y) / (2 * sigma * sigma));
+        rx_lr[i] = y;
```

Comment:
I suggest the following comment, since there is a sign flip lurking in `ldpc_decoder` which has made me trip.

```suggestion
        // The ldpc_decoder performs a sign inversion before passing the LLR data to awgn_bp.
        // The input of awgn_bp is LLRs with the convention that a positive LLR indicates that the
        // bit 0 is more likely.
        rx_lr[i] = y;
```